stack.yaml

```diff
@@ -21,3 +21,7 @@ extra-deps:
 - template-0.2.0.10@sha256:f822de4d34c45bc84b33a61bc112c15fedee6fa6dc414c62b10456395a868f85
 - inline-c-0.9.0.0
 - inline-c-cpp-0.4.0.0
+
+nix:
+  packages: [zlib.dev, zlib.out]
```

Comment:
This shouldn't effect stack builds unless `nix: enabled: true` which is usually set globally for nixos users.